gpuallocator/besteffort_test.go

```diff
@@ -110,6 +110,46 @@ func TestBestEffortAllocate(t *testing.T) {
 			4,
 			[]int{},
 		},
+		{
+			"Required too many devices than available",
```

Comment:
This description does not seem to match the test input. `len(required) == len(available)` here.

Reply:
Fixed in https://github.com/NVIDIA/go-gpuallocator/pull/20#discussion_r1423351783